airflow/www/views.py

```diff
@@ -837,8 +837,8 @@ def log(self):
         if ti:
             host = ti.hostname
             log_loaded = False
-
-            if socket.getfqdn() == host:
+            
+            if conf.get('webserver', 'FORCE_LOCAL_LOGS') or socket.getfqdn() == host:
```

Comment:
I'm not convinced that either of these is the right way to handle this check. Wouldn't it be more straight forward to just check if `os.path.exists(loc)`, and if so, use it?